### PR TITLE
PW-61: Fixed broken links in /docs/tags

### PR DESCRIPTION
**Pull request by abhigyank**
# Summary

* This is a…
    * [x] Bug fix
    * [ ] Feature addition
    * [ ] Refactoring
    * [ ] Minor / simple change (like a typo)
    * [ ] Other
* **Describe this change in 1-2 sentences**:
Point the links to correct ones in Doc and removed link for non existing docs. 
# Problem

* JIRA ticket (_optional_): [PW-61](https://tickets.metabrainz.org/browse/PW-61)

# Solution
Made changes in https://picard.musicbrainz.org/docs/tags/, by fixing the links to correct docs or removing them in case of non-existing.

### CODE REVIEW

File: website/frontend/templates/docs/tags.html [hunk 1]
```diff
@@ -460,9 +460,9 @@ <h1 id="advanced_tags">Advanced Tags</h1>
         </tr>
         <tr>
           <td>arranger</td>
-          <td> <a href="https://musicbrainz.org/doc/index.php?title=Arranger_Relationship_Type&amp;action=edit&amp;redlink=1"
+          <td> <a href="#"
```
Comment (yvanzo) [inline website/frontend/templates/docs/tags.html]: There is some documentation for _arranger_ relationships:
- for a _recording_: https://musicbrainz.org/relationship/22661fb8-cdb7-4f67-8385-b2a8be6c9f0d
- for a _release_: https://musicbrainz.org/relationship/34d5334e-a4c8-4b65-a5f8-bbcc9c81d13d
- for a _work_: https://musicbrainz.org/relationship/d3fd781c-5894-47e2-8c12-86cc0e2c8d08

I’m not sure they are all relevant in the context of Picard tagger, please check that before linking to it.

File: website/frontend/templates/docs/tags.html [hunk 2]
```diff
@@ -559,8 +554,7 @@ <h3 id="last_fm_plus">Last.fm Plus</h3>
 
       <p>
         The
-        <a href="https://musicbrainz.org/doc/index.php?title=LastFMPlus&amp;action=edit&amp;redlink=1" class="new"
-        title="LastFMPlus (page does not exist)">LastFMPlus</a> plugin is a sophisticated plugin that tries
+        <a href="https://musicbrainz.org/doc/MusicBrainz_Picard/Documentation/Plugins/Lastfmplus" title="LastFMPlus">LastFMPlus</a> plugin is a sophisticated plugin that tries
```
Comment (yvanzo) [inline website/frontend/templates/docs/tags.html]: Perfect fit too! 👌

**Comment by yvanzo** [comment]
@abhigyank, sorry, I mistakenly believed it was a GCI task, but it is not. Thanks, btw.

File: website/frontend/templates/docs/tags.html [hunk 3]
```diff
@@ -45,15 +45,15 @@ <h1 id="basic_tags">Basic Tags</h1>
       <table class="table table-bordered table-striped">
         <tr>
           <td>title</td>
-          <td> <a href="https://musicbrainz.org/doc/Track_Title" title="Track Title">Track Title</a></td>
+          <td> <a href="https://musicbrainz.org/doc/Style/Titles" title="Track Title">Track Title</a></td>
```
[the remark this answers is not in the document]
Comment (abhigyank) [reply website/frontend/templates/docs/tags.html]: Ok, I don't think there is any other page for Track title. I'll remove the link.

File: website/frontend/templates/docs/tags.html [hunk 4]
```diff
@@ -45,15 +45,15 @@ <h1 id="basic_tags">Basic Tags</h1>
...
         </tr>
         <tr>
           <td>artist</td>
-          <td> <a href="https://musicbrainz.org/doc/Track_Artist" title="Track Artist">Track Artist</a> Name(s) (string)</td>
+          <td> <a href="https://musicbrainz.org/doc/Artist" title="Track Artist">Track Artist</a> Name(s) (string)</td>
         </tr>
         <tr>
           <td>artists</td>
-          <td> <a href="https://musicbrainz.org/doc/Track_Artist" title="Track Artist">Track Artist</a> Name(s) (multi-value)
+          <td> <a href="https://musicbrainz.org/doc/Artist" title="Track Artist">Track Artist</a> Name(s) (multi-value)
```
[the remark this answers is not in the document]
Comment (abhigyank) [reply website/frontend/templates/docs/tags.html]: The given link isn't about editing guidelines, as above one is. I think this link is appropriate.

File: website/frontend/templates/docs/tags.html (hunk 1 above)
Comment (abhigyank) [reply website/frontend/templates/docs/tags.html]: Yes these are relevant, I'll fix this and make a commit.

File: website/frontend/templates/docs/tags.html (hunk 4 above)
[the remark this answers is not in the document]
Comment (yvanzo) [reply website/frontend/templates/docs/tags.html]: Yes it isn’t about editing guidelines, but it isn’t specifically about _track artist_ either.
However https://musicbrainz.org/doc/Track currently links to it, thus it is probably fine for now.

**Comment by yvanzo** [comment]
Thank you for checking the three relationship types `arranger`. Similarly, can other relationship types `instrumentator`, `remixer`, `producer`, `engineer`, and `mix-DJ` be checked/linked too?

**Comment by zas** [comment]
@abhigyank any update on this ?

**Pull request merged**